src/System.Private.DataContractSerialization/src/System/Xml/PrefixHandle.cs

```diff
@@ -178,8 +179,10 @@ public int CompareTo(PrefixHandle that)
             return GetString().CompareTo(that.GetString());
         }
 
-        private bool Equals2(PrefixHandle prefix2)
+        public bool Equals(PrefixHandle prefix2)
         {
+            if (ReferenceEquals(prefix2, null))
+                return false;
```

Comment:
Oh, nevermind, I see you just move this from the Equals method that already existed.